pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -32,8 +32,12 @@ var (
 	AWSRestrictedLabelDomains = []string{
 		"k8s.aws",
 	}
-	OperatingSystemBottleRocket = "Bottlerocket"
-	OperatingSystemEKSOptimized = "EKSOptimized"
+	AMIFamilyBottlerocket = "Bottlerocket"
+	AMIFamilyEKSOptimized = "EKSOptimized"
```

Comment:
Reminder to make this EKSOptimizedLinux

Reply:
Can't we just support Operating system separately and keep AMIFamily the actual family?  So if AMIFamily was EKSOptimized but the pod requested Windows, we'd just give them windows EKS Optimized?

Reply:
This assumes a certain amount of parity between EKSOptimized. In general, I agree with you.